packages/ui/core/Button/Button.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import styled, {DefaultTheme, withTheme, css} from 'styled-components/native';
+import React, {useContext} from 'react';
+import styled, {DefaultTheme, ThemeContext, css} from 'styled-components/native';
 import {px2ddp} from '@animavita/theme';
```

Comment:
I’m pretty sure that I exported a `useTheme` from theme package

Reply:
Yes, I needed include the styled theme in the hook to use it.